### ISSUE TITLE
Recognize property setters

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#69]
I have (for many reasons) never liked (and more importantly never used) the dot syntax in Objective-C. Due to this, my code look like:

```
[self setName:@"John Doe"];
```

This gets translated into:

```
self.setName("John Doe")
```

However, since it's a synthesized property `name`, this is wrong and should be like this:

```
self.name = "John Doe"
```

I'm not sure how to detect this automatically, but my guess is that if the method name starts with "set" and is followed by a capital letter, it's likely an invocation of a setter...

**Comment by drkameleon** [comment]
Just found its way in my to-do list. :)

---

*P.S. You can't imagine how many of those things I have already thought of, but doing 100 things at the same time I obviously cannot remember all of them. That's why I love GitHub Issues: a great way of keeping things in one place and having the community contribute.*

**Comment by charlieMonroe** [comment]
Thanks! Thanks for the great tool and your responsiveness!

**Comment by drkameleon** [comment]
> Thanks for the great tool and your responsiveness!

If it wasn't for all of you contributing ideas/suggestions/bug-reports/anything, there would have been no "great tool". That's the... fuel behind it! ;-)

**Comment by drkameleon** [comment]
Fixed as of the upcoming 1.9. 

DISCLAIMER: For now, it works only for `[X setY:Z]` type of constructs - simply property setting. `[X setValue:Y forProperty:Z]` or alike will not work.

---

**Example:**

```objective-c
[self setName:@"John Doe"];
```

**Output:**

```swift
self.name = "John Doe"
```
